move_to_initial_position:
  stiffness:
    translational: [400.0, 400.0, 400.0]
    rotational: [20.0, 30.0, 30.0]
# goal: [0.155, 0.3455, 0.050]
#  goal: [0.127, 0.3475, 0.050]
#  goal: [0.155, 0.3228, 0.021]
  goal: [0.127, 0.3228, 0.020]

external_down_movement:
  stiffness:
    translational: [500.0, 500.0, 200.0]
    rotational: [80.0, 80.0, 80.0]
# goal: [0.155, 0.3455, -0.007]
#  goal: [0.127, 0.3475, -0.007]
#  goal: [0.155, 0.3228, -0.007]
  goal: [0.127, 0.3228, -0.007]

spiral:
  stiffness:
    translational: [800.0, 800.0, 250.0]
    rotational: [80.0, 80.0, 80.0]
# goal: [0.155, 0.3455, -0.007]
#  goal: [0.127, 0.3475, -0.007]
#  goal: [0.155, 0.3228, -0.007]
  goal: [0.127, 0.3228, -0.007]




straightening:
  stiffness:
    translational: [600.0, 600.0, 200.0]
    rotational: [80.0, 100.0, 100.0]
# goal: [0.155, 0.3455, -0.009]
#  goal: [0.127, 0.3475, -0.009]
#  goal: [0.155, 0.3228, -0.009]
  goal: [0.127, 0.3228, -0.009]

wiggle:
  stiffness:
    translational: [600.0, 600.0, 400.0]
    rotational: [60.0, 60.0, 60.0]
# goal: [0.155, 0.3455, -0.009]
#  goal: [0.127, 0.3475, -0.009]
#  goal: [0.155, 0.3228, -0.009]
  goal: [0.127, 0.3228, -0.009]


internal_up_movement:
    stiffness:
        translational: [600.0, 600.0, 400.0]
        rotational: [60.0, 60.0, 60.0] 
#   goal: [0.155, 0.3455, 0.075]
#    goal: [0.127, 0.3475, 0.075]
#    goal: [0.155, 0.3228, 0.075]
    goal: [0.127, 0.3228, 0.075]

